CHANGES.rst

```diff
@@ -4,7 +4,9 @@ Changelog
 0.4 (unreleased)
 ----------------
 
-- TBD
+- Add session_factory_from_settings
+
+- Add includeme
```

Comment:
Please note the removed / added support for Python version here as well.  I'd actually prefer to review those changes in a separate PR, but I understand how you go here.